core/store/src/main/java/uk/gov/gchq/gaffer/store/schema/TypeDefinition.java

```diff
@@ -39,7 +39,7 @@
 @JsonFilter(JSONSerialiser.FILTER_FIELDS_BY_NAME)
 public class TypeDefinition {
     private Class<?> clazz;
-    private Serialisation serialiser;
+    private ToBytesSerialiser serialiser;
```

Comment:
All references to ToBytesSerialiser in this class should be Serialiser. In fact all references to ToBytesSerialiser in the entire Store module should be Serialiser. ToBytesSerialiser should only be required in the Accumulo and HBase stores.